msteams-platform/concepts/build-and-test/analyze-your-apps-usage-in-developer-portal.md

```diff
@@ -46,57 +49,189 @@ After you have selected your desired filters, you can explore the following indi
+   The **Usage by region** chart shows the various regions where your app is being used. You can select **View details** for more details on active users in different countries or regions.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-country.png" alt-text="Screenshots shows you app usage by country or region of your published app in Developer Portal."lightbox="../../assets/images/tdp/tdp-region.png":::
+
+ #### Usage intensity
+
+   The **Usage intensity** chart shows the key usage intensity metrics for your app.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-usage-intensity.png" alt-text="Screenshots shows you app usage intensity of your published app in Developer Portal.":::
+
+   | Metric | Definition |
+   | :-----------------------| :------------------------------------------------------------------------------------------------------|
+   | **Median days used per month** | The median number of days in which your app was opened in the last R30 (Rolling 30 days) time period. |
+   | **% of 5+ Days usage** | The percentage of active users who opened or used the app more than five days in the last R30 time period. |
+   | **DAU/MAU** | The ratio of the average number of unique users or tenants who used your app on each day divided by the Monthly active users for the selected R30 time period. |
+
+ #### My app dashboard
+
+   The **My App Dashboard** table shows you the latest R30 (Rolling 30 days) data for each of the metrics under the previous four categories, and the Month over Month change. Use the time picker on the upper right side and select the desired date to view the following:
+
+* Daily R30 data for the last 75 days.
+* End of month R30 data for up to 12 months.
+
+   You can select each of these metric names to see trends over time.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-app-dashboard-view.png" alt-text="Screenshots shows you app capability usage of your published app in Developer Portal."lightbox="../../assets/images/tdp/tdp-app-dashboard-view.png":::
+
+ ### Host product
+
+   The host product page shows you the following app usage analytics:
+
+   * [Usage by host product](#usage-by-host-product)
+   * [Usage by operating system](#usage-by-operating-system)
+   * [Usage by platform](#usage-by-platform)
+
+   The Key Performance Indicator (KPI) cards display metrics such as the host with the most active users and the operating system (OS) with the most active users, based on monthly, weekly, and daily data, in a metric card format.
+
+   :::image type="content" source="../../assets/images/tdp/tdp-host-product.png" alt-text="Screenshots shows you app usage in different host products of your published app in Developer Portal.":::
+
+ #### Usage by host product
+
+   The **Usage by host product** chart shows your app's active usage across various host products, including as **Teams**, **Outlook**, and the **Microsoft 365 app** of monthly, weekly, and daily data.
```

Comment:
Please make sure double check the brand name for Harmony. Since I heard it will going to change to "AI Hub" sometime.

Reply:
@srinidyMSFT , could you please confirm this.

Reply:
@irenewang101 - This is the guidance we got and that Anneliese worked on. Please let me know if there are any POC that I must verify this information with.